packages/react-components/react-portal/src/components/Portal/usePortalMountNodeStyles.styles.ts

```diff
@@ -2,7 +2,8 @@ import { makeStyles } from '@griffel/react';
 
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 23 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28109&commitId=90cb4b542bf9c826edb9c968455603ec425cb5bf&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1921 screenshots unchanged, 0 screenshots with different dimensions and 23 screenshots with visible difference.


  <details><summary><b>unknown</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - Dark Mode.active.chromium.png|2025| Changed|
|InfoButton.default - High Contrast.active.chromium.png|2361| Changed|
|InfoButton.default.active.chromium.png|2042| Changed|
|InfoButton.size.rest.chromium.png|4473| Changed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|11675| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|8656| Changed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|4649| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|4852| Changed|
|Portal.Apply Class Names.should have green border.chromium.png|1885| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|892| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|1229| Changed|
|Tooltip Converged.basic.chromium.png|350| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|453| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|2914| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|7658| Changed|
|Tooltip Converged.inverted.chromium.png|1700| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|5759| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|2674| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|2617| Changed|
|Tooltip Converged.text-wrapping.chromium.png|1556| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|2195| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|2902| Changed|
|Tooltip Converged.withArrow.chromium.png|684| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>